dblookupext/esdtSupply/logsGetter.go

```diff
@@ -87,16 +91,21 @@ func (lg *logsGetter) getTxLog(txHash []byte) (data.LogHandler, bool, error) {
 	return logFromDB, true, nil
 }
 
-func mergeLogsMap(m1, m2 map[string]data.LogHandler) map[string]data.LogHandler {
-	finalMap := make(map[string]data.LogHandler, len(m1)+len(m2))
+func mergeLogsSlices(m1, m2 []indexer.LogData) []indexer.LogData {
+	logsMap := make(map[string]indexer.LogData, len(m1)+len(m2))
```

Comment:
providing a length on a map is not required

Reply:
done